nautobot_design_builder/choices.py

```diff
@@ -15,17 +15,3 @@ class DesignInstanceStatusChoices(ChoiceSet):
         (DISABLED, DISABLED),
         (DECOMMISSIONED, DECOMMISSIONED),
     )
-
-
-class DesignInstanceLiveStateChoices(ChoiceSet):
-    """Status choices for Live State Designs Instance."""
-
-    DEPLOYED = "Deployed"
```

Comment:
to capture the same meaning, could we add these choices to the other `Status` field?

Reply:
Base on our conversation earlier in the week I think I remember that you said we would leave this up to the end user to decide which statuses to have. Am I remembering correctly, or do you still want me to add these back in?

Reply:
yes, you are right. Let's keep some basic ones, and leave the user to extend